ide-backend/IdeSession/Query.hs

```diff
@@ -124,20 +124,23 @@ getDistDir = staticQuery $ return . ideSessionDistDir . ideSessionDir
 -- | Read the current value of one of the source modules.
 getSourceModule :: FilePath -> Query BSL.ByteString
 getSourceModule path = staticQuery $ \IdeStaticInfo{ideSessionDir} ->
-  BSL.readFile $ ideSessionSourceDir ideSessionDir </> path
+  let ideSessionDir' = ideSessionDir </> path
```

Comment:
Related to [this comment], I would have expected this to use `fetchSourceDir` instead of ideSessionDir.